<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>BİÇİMLENDİRME ARAYÜZÜ</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QWidget" name="layoutWidget">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Font:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Büyüklük:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Renk:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QFontComboBox" name="fontComboBox"/>
               </item>
               <item>
                <widget class="QSpinBox" name="sb_Buyukluk">
                 <property name="minimum">
                  <number>6</number>
                 </property>
                 <property name="maximum">
                  <number>50</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cb_Renk">
                 <item>
                  <property name="text">
                   <string>RED</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>GREEN</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>CYAN</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>BLUE</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>YELLOW</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>PURPLE</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>BLACK</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>WHITE</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="chb_Kalin">
               <property name="text">
                <string>BOLD</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chb_Italik">
               <property name="text">
                <string>ITALIC</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>YAZIYI GİR:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txt_Yazi"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Onay">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="YAZDIR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
